pkg/controller/quota/quota.go

```diff
@@ -186,3 +196,12 @@ func isEnforced(req router.Request, namespace string) (bool, error) {
 	}
 	return project.Annotations[labels.ProjectEnforcedQuotaAnnotation] == "true", nil
 }
+
+// replicas returns the number of replicas based on an int32 pointer. If the
+// pointer is nil, it is assumed to be 1.
+func replicas(s *int32) int {
+	if s != nil {
+		return int(*s)
+	}
+	return 1
+}
```

Comment:
nit: This makes me want `z.DereferenceOrDefault`.

Reply:
That could be nice, yeah. There have definitely been times I've wanted something like that. Its probably borderline though.